ui/page.go

```diff
@@ -512,8 +514,26 @@ func (c Container) Layout(gtx layout.Context, w layout.Widget) layout.Dimensions
 	return c.padding.Layout(gtx, w)
 }
 
+var (
+	MaxWidth = unit.Dp(800)
+)
+
 func (page pageCommon) UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
-	return layout.UniformInset(values.MarginPadding24).Layout(gtx, body)
+	width := gtx.Constraints.Max.X
+
+	padding := values.MarginPadding24
+
```

Comment:
a comment should be added here stating that this block of code reduces the width of the content as the window increases in size. 
```suggestion
// reduce the width of the page content as the window increases beyond the maximum width
```